content/pt-br/docs/concepts/cluster-administration/manage-deployment.md

```diff
@@ -0,0 +1,461 @@
+
+```shell
+kubectl get pods -l app=nginx -L tier
+```
+
+```shell
+NAME                        READY     STATUS    RESTARTS   AGE       TIER
+my-nginx-2035384211-j5fhi   1/1       Running   0          23m       fe
+my-nginx-2035384211-u2c7e   1/1       Running   0          23m       fe
+my-nginx-2035384211-u3t6x   1/1       Running   0          23m       fe
+```
+
+Isso exibe todos os pods com "app=nginx", com uma coluna adicional para a _label_ _tier_ (especificada com `-L` ou `--label-columns`).
+
+Para mais informações, por favor visite [_labels_](/docs/concepts/overview/working-with-objects/labels/) e [kubectl _label_](/docs/reference/generated/kubectl/kubectl-commands/#label).
+
+## Atualizando anotações
+
+Às vezes, você gostaria de anexar anotações aos recursos. Anotações são metadados arbitrários não identificadores para serem utilizadas pelos clientes da API como ferramentas, bibliotecas, etc. Isso pode ser feito com `kubectl annotate`. Por exemplo:
+
+```shell
+kubectl annotate pods my-nginx-v4-9gw19 description='my frontend running nginx'
+kubectl get pods my-nginx-v4-9gw19 -o yaml
+```
+
+```shell
+apiVersion: v1
+kind: pod
+metadata:
+  annotations:
+    description: my frontend running nginx
+...
+```
+
+Para mais informações, por favor veja os documentos de [_annotations_](/docs/concepts/overview/working-with-objects/annotations/) e [kubectl _annotate_](/docs/reference/generated/kubectl/kubectl-commands/#annotate).
+
+## Escalando sua aplicação
+
+Quando a carga da sua aplicação aumentar ou diminuir, use `kubectl` para escalar sua aplicação. Por exemplo, para diminuir o número de réplicas do nginx de 3 para 1, faça:
+
+```shell
+kubectl scale deployment/my-nginx --replicas=1
+```
+
+```shell
+deployment.apps/my-nginx scaled
+```
+
+Agora você tem apenas um _pod_ gerenciado pelo _deployment_.
+
+```shell
+kubectl get pods -l app=nginx
+```
+
+```shell
+NAME                        READY     STATUS    RESTARTS   AGE
+my-nginx-2035384211-j5fhi   1/1       Running   0          30m
+```
+
+Para que o sistema escolha automaticamente o número de réplicas nginx de acordo com o necessário, variando de 3 para 1, faça:
+
+```shell
+kubectl autoscale deployment/my-nginx --min=1 --max=3
+```
+
+```shell
+horizontalpodautoscaler.autoscaling/my-nginx autoscaled
+```
+
+Agora as suas réplicas do nginx vão aumentar e diminuir conforme necessário, automaticamente.
+
+Para mais informações, por favor veja os documentos [kubectl _scale_](/docs/reference/generated/kubectl/kubectl-commands/#scale), [kubectl _autoscale_](/docs/reference/generated/kubectl/kubectl-commands/#autoscale) e [horizontal _pod autoscaler_](/docs/tasks/run-application/horizontal-pod-autoscale/).
+
+## Atualização de recursos em execução
+
+Às vezes, é necessário fazer atualizações restritas e sem interrupções nos recursos que você criou.
+
+### kubectl _apply_
+
+É sugerido manter um conjunto de arquivos de configuração no control de origem
```

Comment:
Suggestion: source control in this sentence alludes to source control management (SCM) systems. In this sentence, "sistema de controle de versão de código-fonte" might read better than "control de origem".